spec.md

```diff
@@ -228,6 +228,9 @@ There are three sets of RPCs:
 * **Node Service**: The Node Plugin MUST implement this sets of RPCs.
 
 ```protobuf
+syntax = "proto3";
```

Comment:
These lines will make the original `make` command fail due to Line 138 in `csi.mk`. We probably want to either: 1. remove this and re-generate them in CMake; or 2. fix `csi.mk`; or 3. deprecate `csi.mk` and change the CI test.